api_practice/D:\py\pycharmProjects\api\log_msg.log

```diff
@@ -0,0 +1,1098 @@
+INFO     : 2022-07-17 08:43:35,619 : [server.py : 75] : Started server process [21262]
+INFO     : 2022-07-17 08:43:35,619 : [on.py : 47] : Waiting for application startup.
```

Comment:
Also the api_db.db -> must be deleted. this socialApi application will have no DB. 
(writing comment here since the .db file is not opening in web)

Reply:
api_db.db deleted